WooCommerce/UITestsFoundation/Screens/Login/PasswordScreen.swift

```diff
@@ -29,27 +29,35 @@ public final class PasswordScreen: ScreenObject {
         )
     }
 
-    public func proceedWith(password: String) throws {
-        _ = try tryProceed(password: password)
+    @discardableResult
+    public func enterValidPassword(password: String) throws -> MyStoreScreen {
```

Comment:
What do you think about making `enterValidPassword(password: String)` -> `enterValidPassword()`, and also `enterInvalidPassword(password: String)` -> `enterInvalidPassword()`? 
I think that the function names are quite clear and having them accepting parameter would make it possible to eventually have mistakes like `enterValidPassword(password: "InvalidPW")`.

Reply:
yup that makes sense, though with this change, i've updated it to use the string value `"pw"` instead of referencing from `TestCredentials.password` because of how the folders are structured. but i think it's ok since both valid and invalid passwords are now more consistent. change made in https://github.com/woocommerce/woocommerce-ios/pull/9560/commits/564dee24581d903e478aeacbf00bac8dd282f9f0

Reply:
+1 for removing the input parameter in `enterValidPassword()` to avoid the inconsistency of calling the method with a password that is invalid.